src/components/FileHistory/HistoryRow.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import { Media, Bd, Img } from 'cozy-ui/transpiled/react/Media'
+import { Caption, SubTitle, Bold } from 'cozy-ui/transpiled/react/Text'
+import { Icon, Circle } from 'cozy-ui/transpiled/react'
+import palette from 'cozy-ui/react/palette'
+
+import styles from './styles.styl'
+
+//@todo this component will be moved to UI when polished with an HistoryTimeLine
```

Comment:
By " when polished with an HistoryTimeLine", do you mean that the HistoryRow and Timeline functionality (showing circles and left line) should be separated from the information presented to the right ? If so, I think th ecomment could be a little more explicit, I think that we will soon forget what it means otherwise.

Reply:
In fact we'll not have a TimeLine component. One was enough to manage the vertical line :)

Reply:
I don't understand what do you mean by "One was enough to manage the vertical line :)" ?